src/services/AuthService.tsx

```diff
@@ -64,18 +64,19 @@ export const checkAuthStatusService = () => {
   return authStatus;
 };
 
-export const sendOTP = (phoneNumber: string) => {
+// service to sent the OTP based on the phone number
+export const sendOTP = (phoneNumber: string, registration = 'false') => {
```

Comment:
any reason for going with string instead of boolean for the registration param?

Reply:
The backend needs it that way,